pytest_doctestplus/plugin.py

```diff
@@ -10,6 +10,8 @@
 import sys
 import warnings
 
+from packaging.version import Version
```

Comment:
You'll want to add `packaging` to `install_requires` in setup.py.  This is working now because `pytest` also depends on `packaging` and installs it.

Reply:
I think `packaging` is vendored by `setuptools`

Reply:
It is, but it doesn't not have the same namespace.  You then have to import it from `setuptools.extern.version`.